pkg/kubectl/cmd/util/factory.go

```diff
@@ -67,8 +70,14 @@ type Factory interface {
 // Generally provides discovery, negotiation, and no-dep calls.
 // TODO The polymorphic calls probably deserve their own interface.
 type ClientAccessFactory interface {
+	// Returns a client.Config for accessing the Kubernetes server.
```

Comment:
Embed `RESTClientGetter` here rather than repeating methods, which might lead to out of sync state.